src/standalone/Soil/rre.jl

```diff
@@ -92,6 +92,7 @@ function RichardsModel{FT}(;
         subsurface_source = subsurface_runoff_source(top_bc.runoff)
         sources = append_source(subsurface_source, sources)
     end
+    @assert ~lateral_flow
```

Comment:
```suggestion
    @assert !lateral_flow
```

`!` is for logical operations, `~` is for bitwise operations